src/rmodels.c

```diff
@@ -5110,7 +5110,7 @@ static Model LoadGLTF(const char *fileName)
                 {
                     // The primitive actually keeps the pointer to the corresponding material,
                     // raylib instead assigns to the mesh the by its index, as loaded in model.materials array
-                    // To get the index, we check if material pointers match and we assign the corresponding index,
+                    // To get the index, we check if material pointers match, and we assign the corresponding index,
```

Comment:
models.c line 5113, again, it seems better without the ",".  Again, it is about a combined act that has the result.  I don't claim authority though :).